rust_src/src/cmds.rs

```diff
@@ -1,18 +1,66 @@
 //! Commands
 
 use remacs_macros::lisp_fn;
-use remacs_sys::EmacsInt;
+use remacs_sys::{EmacsInt, Fline_end_position};
```

Comment:
Types in one block, functions in the next. This is the pattern of the recent commits to try and reduce lines of conflict.